db/mongodb.js

```diff
@@ -1,25 +1,45 @@
 const mongoose = require('mongoose');
+// var autoIncrement = require("mongodb-autoincrement");
 
-mongoose.connect('mongodb://Miken:1234@ds263138.mlab.com:63138/restaurants_reviews');
+
+mongoose.connect('mongodb://localhost/Restaurants_Reviews')
+
+// mongoose.plugin(autoIncrement.mongoosePlugin);
 
 const restaurantSchema = mongoose.Schema({
   restaurantId: Number,
   restaurantName: String,
-  restaurantReviews: [
-    {
-      username: String,
-      city: String,
-      dinedDate: Date,
-      rating: Number,
-      review: String,
-    },
-  ],
+  username: String,
+  city: String,
+  dinedDate: String,
+  rating: Number,
+  review: [],
 });
+  // restaurantId: Number,
+  // restaurantName: String,
+  // restaurantReviews: [
+  //   {
+  //     username: String,
+  //     city: String,
+  //     dinedDate: String,
+  //     rating: Number,
+  //     review: String,
+  //   },
+  // ],
+// const restaurantSchema = mongoose.Schema({
+//       id: Number,
+//       username: String,
+//       city: String,
+//       dinedDate: String,
+//       rating: Number,
+//       review: String
+//   });
+
 
 const Restaurant = mongoose.model('Restaurant', restaurantSchema);
 
 function insertOne(restaurant, callback) {
-  Restaurant.create(restaurant, callback);
+  Restaurant.create(restaurant, callback).exec();
```

Comment:
You could probably take out this .exec()

Reply:
good catch its pointless